Info.plist

```diff
@@ -54,6 +54,8 @@
 	<string>$(POCKET_API_BASE_URL)</string>
 	<key>PocketAPIConsumerKey</key>
 	<string>$(POCKET_API_CONSUMER_KEY)</string>
+	<key>PocketAPIConsumerKeyPad</key>
+	<string>$(POCKET_API_CONSUMER_KEY_PAD)</string>
```

Comment:
Note add to bitrise env variables

Reply:
@bassrock I believe we discussed this live and determined we would perform no other action at this time.

Reply:
@dskuza right, but would still like to understand what the experience is going to do so it can be communicated.

Reply:
@bassrock Fair, I missed the second part of your comment somehow 😅 . I can test that path, but I believe things will just seem to … not … be … working? Off to verification land…

Reply:
@bassrock Looks like things like pull-to-refresh on My List fail and spin, and empties the list. Looks like at least _one_ resolution would be communicate to iPad users that if they're seeing issues to log out and log back in (which I still think is okay since we're in beta - this isn't something that would affect a production release).

Reply:
@dskuza cool, yep given this would make the next beta if merged today, and @nzeltzer will be emailing folks, let's post that in the slack channel for the team to know.